docs/.vuepress/components/BuildPage.vue

```diff
@@ -215,132 +126,126 @@ export default {
     langPath: function() {
       return translate('path', this.$lang)
     }
+  },
+  data: function() {
+    return {
+      templates: [
+        {
+          title: this.translateString('page-build-hello-world-title'),
```

Comment:
Same issue here as in HomePage.vue, you'll need to bring this into the template.

Reply:
Resolved by moving this `computed:`